lucene/core/src/java/org/apache/lucene/codecs/lucene80/Lucene80DocValuesProducer.java

```diff
@@ -370,6 +378,11 @@ public void close() throws IOException {
     long termsIndexLength;
     long termsIndexAddressesOffset;
     long termsIndexAddressesLength;
+
+    boolean compressed;
+    // Reserved for support other compressors.
+    int compressorCode;
```

Comment:
I'm not sure we need this compressorCode field. The current code does not use it and this future seems too fuzzy.

Reply:
will remove this..just thought we could support more types of compression algorithms here...